static/menu.js

```diff
@@ -0,0 +1,6 @@
+let menu = document.getElementById("collapsible-menu");
+menu.classList = "collapse navbar-collapse";
```

Comment:
I’m not sure if reassigning `classList` is blessed by the DOM Standard. If we want to replace the whole list anyways, then `className` is probably the better property to use. Alternatively, treating it as a proper list would be more readable, I think (making it more obvious that we’re not just adding classes):

```js
menu.classList.remove('nojs');
menu.classList.add('collapse', 'navbar-collapse');
```